org.eclipse.lsp4e/src/org/eclipse/lsp4e/LanguageServers.java

```diff
@@ -172,7 +171,8 @@ public <T> CompletableFuture<Optional<T>> computeFirst(BiFunction<? super Langua
 				executeOnServers(queryLS)
 				.map(lsRequest -> {
 					CompletableFuture<T> populateFuture = lsRequest.thenApply(t -> {
-						if (!isEmpty(t)) { // some LS methods return null objects when they have nothing to report, and some return an empty List
```

Comment:
could we undo this part of the PR and keep the comment and the `isEmpty` method?